src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviInspIRCdIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'c': return __tr2qs("c: Prevent CTCPs"); break;
+		case 'd': return __tr2qs("d: Deaf"); break;
+		case 'g': return __tr2qs("g: Server side ignore"); break;
+		case 'h': return __tr2qs("h: Available for help (HelpOp)"); break;
+		case 'k': return __tr2qs("k: Network service"); break;
+		case 'r': return __tr2qs("r: Registered"); break;
+		case 'x': return __tr2qs("x: Host hidden"); break;
+		case 'B': return __tr2qs("B: Marks as being a bot"); break;
+		case 'G': return __tr2qs("G: Censor bad words"); break;
+		case 'H': return __tr2qs("H: Hide oper status"); break;
+		case 'I': return __tr2qs("I: Channels hidden from WHOIS"); break;
+		case 'Q': return __tr2qs("Q: Marks an IRC Operator as invisible from user lists"); break;
+		case 'R': return __tr2qs("R: Receive private messages from registered nicks only"); break;
+		case 'S': return __tr2qs("S: Strip colors out of private messages"); break;
+		case 'W': return __tr2qs("W: Recipient for notices on WHOIS"); break;
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviCritenIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'C': return __tr2qs("C: Can see CHATOPS notices"); break;
+		case 'D': return __tr2qs("D: Has seen DCCALLOW warning message"); break;
+		case 'F': return __tr2qs("F: Recipient for far connect/quit notices"); break;
+		case 'P': return __tr2qs("P: Services administrator"); break;
+		case 'S': return __tr2qs("S: Network service"); break;
+		case 'W': return __tr2qs("W: Recipient for notices on WHOIS"); break;
+		case 'Z': return __tr2qs("Z: Services root administrator"); break;
+		case 'a': return __tr2qs("a: Services operator"); break;
+		case 'e': return __tr2qs("e: Recipient for blocked DCC notices"); break;
+		case 'f': return __tr2qs("f: Recipient for exess flood notices"); break;
+		case 'g': return __tr2qs("g: Recipient for GLOBOPS notices"); break;
+		case 'h': return __tr2qs("h: Available for help (HelpOp)"); break;
+		case 'm': return __tr2qs("m: Recipient for spam warning notices"); break;
+		case 'n': return __tr2qs("n: Recipient for NETINFO and routing notices"); break;
+		case 'p': return __tr2qs("p: Protected irc operator"); break;
+		case 'r': return __tr2qs("r: Registered"); break;
+		case 'x': return __tr2qs("x: Host hidden"); break;
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviIrcdRatboxIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'C': return __tr2qs("C: Prevent CTCPs"); break;
+		case 'D': return __tr2qs("D: Deaf"); break;
+		case 'Z': return __tr2qs("Z: Recipient for oper spy notices"); break;
+		case 'a': return __tr2qs("a: Server administrator"); break;
```

Comment:
I've changed everything to administrator where applicable, the only two places I'm not going to change it are `Server/Network/Tech Admin only channel` because `Tech Admin` is an Unreal/Criten oper type, and `Co-Admin` because that's what Unreal officially refers to it as.